Specs/addDefaultMatchers.js

```diff
@@ -269,6 +269,22 @@ define([
                 };
             },
 
+            toPickNearbyPrimitive : function(util, customEqualityTesters) {
+                return {
+                    compare : function(actual, expected, config) {
+                        return pickPrimitiveEqualsWrapper(actual, expected, config.x, config.y, config.size);
+                    }
+                };
+            },
+
+            notToPickNearbyPrimitive : function(util, customEqualityTesters) {
+                return {
+                    compare : function(actual, config) {
+                        return pickPrimitiveEqualsWrapper(actual, undefined, config.x, config.y, config.size);
+                    }
+                };
+            },
```

Comment:
Instead of adding these two perhaps just add the functionality to `toPickPrimitive` and `notToPick`.